src/components/layout/Header.tsx

```diff
@@ -323,27 +265,7 @@ export const Header: React.FC = (props) => {
                     </DropdownButton>
                   }
                   dropdownPosition={DropdownPosition.center}
-                  items={[
-                    <WalletDropdownContents key={'wallet_dopdown'}>
-                      <WalletDropdownRow>
-                        <Metamask />
-                      </WalletDropdownRow>
-                      <WalletDropdownRow>{address && shortenAddress(address)}</WalletDropdownRow>
-                      {address && (
-                        <WalletDropdownRow>
-                          <CopyButton copyValue={address} />
-                          <ExternalLink
-                            href={`${currentChain.blockExplorerUrls}/address/${address}`}
-                            target="_blank"
-                          >
-                            <LinkSVG />
-                          </ExternalLink>
-                        </WalletDropdownRow>
-                      )}
-                      <WalletButton>Change wallet</WalletButton>
-                      <WalletButton onClick={disconnectWallet}>Disconnect</WalletButton>
-                    </WalletDropdownContents>,
-                  ]}
+                  items={[<ChangeWalletMenu key={'wallet_dopdown'} />]}
```

Comment:
Is this code part of https://github.com/AelinXYZ/aelin-frontend-v2/issues/549? Would be good to send another PR that includes this improvement.